lib/keen/styles/variables.scss

```diff
@@ -34,13 +34,13 @@ $ui-input-margin-bottom: rem(16px) !default;
 
 // Input label
 $ui-input-label-color: $secondary-text-color !default;
```

Comment:
Would be good to file a follow up issue to remove any SCSS variables that rely on theming SCSS variables, as they have the potential to break our theming.

In general, this should only be an issue inside our vendored Keen components (and could be part of a broader push to fully integrate/get rid of our vendored keen components entirely).

Reply:
@rtibbles Some time ago, we agreed with @marcellamaki that I will work on creating a whole milestone and filling it with exactly these kinds of issues after the main rebrand push

Reply:
No problem for @LianaHarris360 to open a follow-up now, I just thought I would spare her because it's a pervasive issue in the whole KDS which will be included in the milestone more widely